target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds

```diff
@@ -8,6 +8,15 @@ board=$(board_name)
 boardname="${board##*,}"
 
 case "$board" in
+mikrotik,routerboard-952ui-5ac2nd)
+	ucidef_set_led_timer "user" "USER" "mikrotik:green:user" "1000" "1000"
```

Comment:
I'm not a fan of ucidef_set_led_timer, and it's almost never used nowadays, as "blinking" indicates boot or similar.
I'd just drop this line.